src/ui/MultipleFilesViewer/index.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { ReactElement, useContext, useEffect, useState } from 'react';
+import './index.scss';
+import { LocalizationContext } from '../../lib/LocalizationContext';
+import Avatar from '../Avatar';
+import { isImage, isSupportedFileView } from '../../utils';
+import Label, { LabelTypography, LabelColors } from '../Label';
+import Icon, { IconColors, IconTypes } from '../Icon';
+
+const BUTTON_ICON_SIDE_LENGTH = '24px';
+
+export interface FileInfo {
+  name: string;
+  type: string;
+  url: string;
+}
+interface Props {
+  profileUrl: string;
+  nickname: string;
+  fileInfoList: FileInfo[];
```

Comment:
If we expect the array of FileInfo here, this component will be used for only MFM
How about getting an array of Files?

Reply:
No this is new FileInfo interface I added in this PR